content/reference-transaction-format.md

```diff
@@ -1374,6 +1374,7 @@ These codes indicate that the transaction failed and was not included in a ledge
 | `tefEXCEPTION`           | While processing the transaction, the server entered an unexpected state. This may be caused by unexpected inputs, for example if the binary data for the transaction is grossly malformed. If you can reproduce this error, please [report an issue](https://github.com/ripple/rippled/issues) to get it fixed. |
 | `tefFAILURE`             | Unspecified failure in applying the transaction.    |
 | `tefINTERNAL`            | When trying to apply the transaction, the server entered an unexpected state. If you can reproduce this error, please [report an issue](https://github.com/ripple/rippled/issues) to get it fixed. |
+| `tefINVARIANT_FAILED`   | An invariant check failed when trying to claim the [transaction cost](concept-transaction-cost.html). Requires the [EnforceInvariants amendment](concept-amendments.html#enforceinvariants). If you can reproduce this error, please [report an issue](https://github.com/ripple/rippled/issues). |
```

Comment:
I think "error" is clearer here. Yes, technically it's a "result code" not an "error" but it's a failure result, which is basically an error, so that's more intuitive.